sdk/synapse/Azure.Analytics.Synapse.AccessControl/README.md

```diff
@@ -59,26 +59,37 @@ The way you control access to Synapse resources is to create role assignments. A
 The Azure.Analytics.Synapse.AccessControl package supports synchronous and asynchronous APIs. The following section covers some of the most common Azure Synapse Analytics access control related tasks:
 
 ### Role assignment examples
+* [Create access control client](#create-access-control-client)
 * [Create a role assignment](#create-a-role-assignment)
 * [Retrieve a role assignment](#retrieve-a-role-assignment)
 * [List role assignments](#list-role-assignments)
 * [Delete a role assignment](#delete-a-role-assignment)
 
+### Create access control client
+
+`CreateAccessControlClient` creates a role assignment.
+
+```C# Snippet:CreateAccessControlClient
+// Replace the string below with your actual workspace url.
+string workspaceUrl = "<my-workspace-url>";
+AccessControlClient client = new AccessControlClient(endpoint: new Uri(workspaceUrl), credential: new DefaultAzureCredential());
+```
+
 ### Create a role assignment
 
 `CreateRoleAssignment` creates a role assignment.
 
 ```C# Snippet:CreateRoleAssignment
 RoleAssignmentOptions options = new RoleAssignmentOptions(sqlAdminRoleId, principalId);
-RoleAssignmentDetails roleAssignment = client.CreateRoleAssignment(options);
+string assignmentId = client.CreateRoleAssignment(options).Value.Id;
 ```
 
 ### Retrieve a role assignment
 
 `GetRoleAssignmentById` retrieves a role assignment by the given principal ID.
 
 ```C# Snippet:RetrieveRoleAssignment
-RoleAssignmentDetails roleAssignment = client.GetRoleAssignmentById(principalId);
+RoleAssignmentDetails roleAssignment = client.GetRoleAssignmentById(assignmentId);
```

Comment:
Should line 89 be updated too? or should the variable to use be `principalID` like before